### ISSUE TITLE
Normalization layer

### ISSUE THREAD

**Issue by zwep** [junyanz/pytorch-CycleGAN-and-pix2pix#1005]
Hi,

is there a reason why you can re-use the normalization layer across the model? I thought that is better to initialize a new norm-layer each time.
I see that you can initiate a batch-norm layer without any tracking of statistics. So in that case, everything goes fine.
But there is an option to do the tracking of the statistics. In that case, doesnt the model go awry?

**Comment by junyanz** [comment]
We use different normalization layers across the model. The [get_norm_layer](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/networks.py#L18) function returns a function object, rather than the layer instance. You can modify this function to track statistics.

**Comment by zwep** [comment]
Ah yeah I see, thanks. 

I asked this because when I used this model in torch's .eval() mode, I get really strange results. I've read that this could happen if you use the same layer across the model, but that is not the case then.
So hopefully if I put the tracking statistics off it behaves normally.

**Comment by junyanz** [comment]
Yes. The results look better without eval() mode.
